README.md

```diff
@@ -4,6 +4,26 @@ This project is part of ASP.NET Core. You can find samples, documentation and ge
 
 Use https://github.com/aspnet/AspNetCore/issues for issues with this project.
 
+## Build this library
+
+There are multiple ways to build this library
+
+* Use [vcpkg](https://github.com/microsoft/vcpkg) and install the library with `vcpkg install microsoft-signalr`
+* Build from [command line](#command-line-build)
+* Build in Visual Studio by selecting the "Open a local folder" option and selecting the repository root folder
```

Comment:
What about prereqs? When I tried this without any C++ components installed, VS didn't load any projects or know what to build.

Reply:
FWIW, the command line instructions fail on windows without VS C++ components installed:

```
C:\dev\aspnet\SignalR-Client-Cpp master ≡ ❯ .\submodules\vcpkg\bootstrap-vcpkg.bat
Could not find MSBuild version with C++ support. VS2015, VS2017, or VS2019 (with C++) needs to be installed.
At C:\dev\aspnet\SignalR-Client-Cpp\submodules\vcpkg\scripts\bootstrap.ps1:195 char:5
+     throw "Could not find MSBuild version with C++ support. VS2015, V ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Could not find ...o be installed.:String) [], RuntimeException
    + FullyQualifiedErrorId : Could not find MSBuild version with C++ support. VS2015, VS2017, or VS2019 (with C++) needs to be installed.
```

Reply:
Good point, do you know the exact requirements?

Reply:
Enabling "Desktop Development with C++" workload and installing cmake was enoughto make the command line instructions work for me. I'm not sure exactly which individual components were needed, but I think telling people to install the workload is sufficient.

![image](https://user-images.githubusercontent.com/54385/103961245-f19b3100-5108-11eb-9cd6-24d639d5d2de.png)